04/app.js

```diff
@@ -5,6 +5,34 @@ function init() {
     setBoxShadow(boxElement, '#000000');
 }
 
+const colorEl = document.querySelector('input[type="color"]')
+const rangeEl = document.querySelector('input[type="range"]')
+const boxElement = document.querySelector('div')
+
+colorEl.addEventListener('change', changeColor);
+
+function changeColor() {
+    const color = colorEl.value;
+    setBoxShadow(boxElement, color, rangeEl.value);
```

Comment:
Przy zmianie koloru należy utrzymać przeźroczystość.
Obecnie to nie działa. Zobacz, że w funkcji niżę trochę inaczej obliczasz opacity.